custom/icds/translations/integrations/transifex.py

```diff
@@ -83,3 +93,64 @@ def _cleanup(self):
         for resource_name, filepath in self.po_file_generator.generated_files:
             if os.path.exists(filepath):
                 os.remove(filepath)
+
+    def _get_resource_slugs_for_version(self, version):
```

Comment:
It looks like `version` is always just `self.version`.  Would it be possible to make a memoized property that's just `resource_slugs`?  Then you wouldn't need to pass it around as much.

Reply:
Ya, removed passing version argument.
> memoized property that's just resource_slugs

sorry, could not get this.

Reply:
haha, it took me a minute to decipher my own words 3 days later here.  I meant that you could replace
```python
def _get_resource_slugs_for_version(self, version):
```
with
```python
@property
@memoized
def resource_slugs(self):
```